lib/Models/RegionMapping.js

```diff
@@ -590,14 +594,16 @@ function addDescriptionAndProperties(regionMapping, regionIndices, regionImagery
                 if (defined(cRowObject)) {
                     cRowObject.string._terria_columnAliases = columnAliases;
                     cRowObject.string._terria_numericalProperties = cRowObject.number;
-                    imageryLayerFeatureInfo.properties = cRowObject.string;
+                    imageryLayerFeatureInfo.properties = Object.assign({}, feature.properties, cRowObject.string);
```

Comment:
We can't use `Object.assign` because it's not available in all browsers and it's inappropriate for a library to globally polyfill something.  Use Cesium's `clone` instead.

Reply:
or Cesium's `combine`.